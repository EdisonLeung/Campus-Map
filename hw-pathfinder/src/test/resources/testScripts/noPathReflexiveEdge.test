# tests pathfinding on a node with children in a reflexive pattern and
# no path. Tests that algorithm will not create infinite loop

# Create Graph
CreateGraph noPath

# add nodes
AddNode noPath a
AddNode noPath b
AddNode noPath c

# add edges
AddEdge noPath a b 5.0
AddEdge noPath b a 5.0

# test pathfinding
FindPath noPath a b

FindPath noPath a c