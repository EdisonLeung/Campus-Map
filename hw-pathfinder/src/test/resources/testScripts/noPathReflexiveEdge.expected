# tests pathfinding on a node with children in a reflexive pattern and
# no path. Tests that algorithm will not create infinite loop

# Create Graph
created graph noPath

# add nodes
added node a to noPath
added node b to noPath
added node c to noPath

# add edges
added edge 5.000 from a to b in noPath
added edge 5.000 from b to a in noPath

# test pathfinding
path from a to b:
a to b with weight 5.000
total cost: 5.000

path from a to c:
no path found